esp/platform/application_config.cpp

```diff
@@ -436,6 +437,15 @@ void setLDAPSecurityInWSAccess(IPropertyTree *legacyEsp, IPropertyTree *legacyLd
     }
 }
 
+// Copy trace flags from appEsp to legacyEsp. The source is expected to be an application's `esp`
+// configuration object. The destination is expected to be the `EspProcess` element.
+inline static void addTraceFlags(IPropertyTree *legacyEsp, IPropertyTree *appEsp)
+{
+    IPropertyTree *traceFlags = appEsp->queryPropTree(propTraceFlags);
+    if (traceFlags)
+        legacyEsp->setPropTree(propTraceFlags, traceFlags);
```

Comment:
This is missing a LINK().  I think it is likely to cause a crash when esp terminates.

Reply:
Agreed.

Reply:
@ghalliday made and squashed the one line change.